src/plugins/mongodb/mongodb_instance_config.cpp

```diff
@@ -238,21 +242,25 @@ bool
 MongoDBInstanceConfig::check_alive()
 {
 	try {
-		std::shared_ptr<mongo::DBClientConnection> client =
-		  std::make_shared<mongo::DBClientConnection>();
-		std::string        errmsg;
-		mongo::HostAndPort hostport("localhost", port_);
-		if (!client->connect(hostport, errmsg)) {
-			return false;
-		}
-		mongo::BSONObj cmd(BSON("isMaster" << 1));
-		mongo::BSONObj reply;
-		bool           ok = client->runCommand("admin", cmd, reply);
+		std::ostringstream hostport;
```

Comment:
Isn't a stringstream a little overkill here, if a simple string would do:
```
std::string hostport{"mongodb://localhost:" + std::to_string(port_)};
```

Reply:
Yes, good point. I also directly created the client on the stack instead of creating a client on the heap with `std::make_shared`.